lib/auth.js

```diff
@@ -49,32 +48,18 @@ module.exports = () => {
   let timer = null;
   const { db } = application;
 
-  const fillPool = () => {
-    const need = SANDBOX_POOL - pool.length;
-    for (let i = 0; i < need; i++) {
-      pool.push(application.createSandbox());
-    }
-  };
-
-  const getSandbox = () => {
-    if (timer === null) {
-      timer = setTimeout(() => {
-        fillPool();
-        timer = null;
-      }, 0);
-    }
-    if (pool.length > 0) return pool.pop();
-    return application.createSandbox();
-  };
-
   const save = (token, context) => {
     const data = JSON.stringify(context);
     db.update('Session', { data }, { token });
   };
 
   class Session {
-    constructor(token, sandbox, contextData = { token }) {
+    constructor(token, contextData = {}) {
```

Comment:
Removing token in stored sessions is a different issue, don't mix all changes to single commit, see https://github.com/metarhia/impress/pull/1241 there are 6 commits